elfpy/pricing_models/yieldspace.py

```diff
@@ -566,16 +546,13 @@ def calc_out_given_in(
                 share_price=share_price,
                 init_share_price=init_share_price,
             )
-            # The fees are calculated as the difference between the bonds
-            # received without slippage and the base paid times the fee
-            # percentage. This can also be expressed as:
-            #
-            # ((1 / p) - 1) * phi * c * dz
-            fee = ((1 / spot_price) - 1) * trade_fee_percent * share_price * d_shares
+            curve_fee = abs(in_amount - without_fee_or_slippage) * trade_fee_percent
```

Comment:
shouldn't this be `curve_fee = abs(without_fee_or_slippage - in_amount) * trade_fee_percent`?

Reply:
Also, why do you need `abs`?

Reply:
works correctly as suggested, implemented in 949ae8faae83acb3c18cc21996299a335c02de14